### PR TITLE
Fix/eco 1986 migrate to latest demoshop

### PR DESCRIPTION
**Pull request by profuel**
- refactored JS & CSS

### CODE REVIEW

**Comment by profuel** [comment]
@tolerants please review as well

**Review by alex-galych — APPROVED** [review]

File: assets/Yves/js/module/checkout-widgets/css/amazon-pay-payment-widgets.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#amazonpayAddressBookWidgetDiv {
```
Comment (tolerants) [inline assets/Yves/js/module/checkout-widgets/css/amazon-pay-payment-widgets.css]: Can you please change these id's? Just replace a "Div" with "Container".

File: assets/Yves/js/module/checkout-widgets/address-book.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+new OffAmazonPayments.Widgets.AddressBook({
+    sellerId: amazonpayConfig.sellerId,
+    scope: 'profile payments:widget payments:shipping_address payments:billing_address',
+    language: amazonpayConfig.locale,
+    onOrderReferenceCreate: function(orderReference) {
+        // Here is where you can grab the Order Reference ID.
+        var aoid = orderReference.getAmazonOrderReferenceId();
+
+        $(function() {
```
Comment (tolerants) [inline assets/Yves/js/module/checkout-widgets/address-book.js:11]: I'm not sure why do you need document.ready here?

File: assets/Yves/js/module/checkout-widgets/address-book.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            $.post(
+                amazonpayConfig.setOrderReferenceUrl,
+                {'reference_id': aoid}
+            );
+        });
+    },
+    onAddressSelect: function(orderReference) {
+        $(function() {
```
Comment (tolerants) [inline assets/Yves/js/module/checkout-widgets/address-book.js:19]: I'm not sure why do you need document.ready here?

File: assets/Yves/js/module/pay-button/index.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+            window.location = amazonpayConfig.redirectUrl + '?'
+                + 'reference_id=' + referenceId
+                + "&access_token=" + addressConsentToken;
+        }
+    });
+};
+
+if (typeof window.errorHandler !== 'function') {
+    window.errorHandler = function (message) {
+        alert(message);
```
Comment (tolerants) [inline assets/Yves/js/module/pay-button/index.js:48]: Are we ok with alerting of error messages?

**Review by tolerants — CHANGES_REQUESTED** [review]

File: assets/Yves/js/module/checkout-widgets/address-book.js (hunk 2 above)
Comment (profuel) [reply assets/Yves/js/module/checkout-widgets/address-book.js:11]: right, not needed here

File: assets/Yves/js/module/pay-button/index.js (hunk 4 above)
Comment (profuel) [reply assets/Yves/js/module/pay-button/index.js:48]: This is edge case when something goes wrong with amazon pay setup. Should never happen

**Review by tolerants — APPROVED** [review]

**Pull request merged**